src/controllers/users-controller.js

```diff
@@ -64,7 +64,7 @@ module.exports = {
     ctx.assert(
       _.isObject(body.user) && body.user.email && body.user.password,
       422,
-      new ValidationError(["is invalid"], "", "email or password"),
+      new ValidationError(["malformed request"], "", "email or password"),
```

Comment:
where did you find the info, that message should be changed?

[here](https://github.com/gothinkster/node-express-realworld-example-app/blob/ba04b70c31af81ca7935096740a6e083563b3a4a/config/passport.js#L12) I see the same message as in this repo

Reply:
Please see https://github.com/gothinkster/koa-knex-realworld-example/pull/12#issuecomment-513127294